code/modules/mob/living/brain/posibrain.dm

```diff
@@ -181,6 +229,12 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 	brainmob.real_name = brainmob.name
 	brainmob.forceMove(src)
 	brainmob.container = src
+
+	//If we are on the station level, add it to the list of available posibrains.
+	var/datum/job/cyborg/posibrain/pj = SSjob.GetJob(JOB_NAME_POSIBRAIN)
+	pj.check_add_posi_slot(src)
+
+
```

Comment:
```suggestion
```

Reply:
This is required, why are you suggesting that I remove it?

Reply:
I am suggesting to remove the double whitespace